OBAKit/Location/OBARegionStorage.m

```diff
@@ -84,14 +90,14 @@ - (void)writeRegionsToDisk:(NSArray<OBARegionV2*>*)regions {
         return nil;
     }
 
-    NSArray<OBARegionV2*>* regions = nil;
-
-    @try {
-        regions = [NSKeyedUnarchiver unarchiveObjectWithFile:OBARegionStorage.localRegionsFilePath];
-    } @catch (NSException *exception) {
-        DDLogError(@"Failed to unarchive persisted regions: %@", exception);
+    NSData *objectData = [NSData dataWithContentsOfFile:OBARegionStorage.localRegionsFilePath];
+    NSError *error;
+    NSArray *regions = [NSKeyedUnarchiver unarchiveTopLevelObjectFor:objectData error:&error];
+    
+    if (error) {
+        DDLogError(@"Failed to unarchive persisted regions: %@", error);
     }
-
+    
```

Comment:
make sure you enable the two "automatically trim trailing whitespace" options in Xcode's preferences. 

<img width="912" alt="image" src="https://user-images.githubusercontent.com/2254/73140267-f2d46a80-402b-11ea-9448-50e422b14c6f.png">